runtime/src/execute_cost_table.rs

```diff
@@ -106,6 +106,17 @@ impl ExecuteCostTable {
             }
         }
 
+        // Logs events of unusual input `value` or `aggregated.ema`
+        const MAX_COST: u64 = 200_000;
```

Comment:
why 200k?

Reply:
200K is current default max (https://github.com/solana-labs/solana/blob/master/program-runtime/src/compute_budget.rs#L79), so any instruction be calculated have more than that are suspicious.

Reply:
Right, but should we not prepare for the future? https://github.com/solana-labs/solana/blob/5c722519cf7b41095fb89c27eafad9a7cf1188d0/program-runtime/src/compute_budget.rs#L76-L80

Let's make that a `pub` function and reuse it here

Reply:
Not too convinced the logic used there in picking max_units is shared here. But with reversion of ema, we don't really have use of this log line anymore. I'll just remove it to keep this patch PR clean.

Reply:
kk.  is there a new place we can add a metric for unexpected values and hook up alerts?  that we didn't see this issue coming at all is pretty disturbing

Reply:
A metrics counter that increases when calculation went out-of-band would be useful. EMA code should have one of these. I'll add a note to add one of these to it when issue is identified and EMA to be restored.

Reply:
My concern is that the current code may be similarly broken, just slower to present or more resilient to degenerate input